migrations/extras/extras.ts

```diff
@@ -6,7 +6,7 @@ export const linkableModelApiKeys = [
   'academy_course',
   'blog_post',
   'changelog_entry',
-  'customer_story',
+  'casual_chat',
```

Comment:
These are Model API keys! We do not need to change them if we are only changing the URLs